extensions/wikia/AdEngine/js/config/desktop.js

```diff
@@ -12,6 +12,7 @@ define('ext.wikia.adEngine.config.desktop', [
 	'ext.wikia.adEngine.provider.evolve',
 	'ext.wikia.adEngine.provider.liftium',
```

Comment:
Note to Ad Engineering: liftium is placed wrongly. Next time we touch `config/desktop.js` we could move it down so we keep alphabetical order.